src/Controller/HomeController.php

```diff
@@ -5,12 +5,41 @@
 
 use XdebugDotOrg\Model\Download;
 use XdebugDotOrg\Model\Downloads;
+use XdebugDotOrg\Model\Supporters;
 
 class HomeController
 {
+	public static function getSupportersModel() : Supporters
+	{
+		$d = \dir( '../data/reports' );
+
+		$files = [];
+
+		while ( false !== ( $entry = $d->read() ) ) {
+			if (preg_match( '@^20[0-9][0-9]-[01][0-9]\.txt$@', $entry, $m)) {
+				$files[] = $entry;
+			}
+		}
+
+		\rsort($files);
+
+		return new Supporters(
+			self::get_supporters()
+		);
+	}
+
 	public static function index() : HtmlResponse
 	{
-		return new HtmlResponse(null, 'home/index.php');
+		return new HtmlResponse(
+			\XdebugDotOrg\Core\ContentsCache::fetchModel(
+				SupportLog::class,
```

Comment:
This requires a `use XdebugDotOrg\Model\SupportLog;`

Reply:
Shouldn't the code be `Supporters::class` instead?

Reply:
Yes